app/models/super_good/solidus_taxjar/order_transaction.rb

```diff
@@ -0,0 +1,7 @@
+module SuperGood
+  module SolidusTaxjar
+    class OrderTransaction < ActiveRecord::Base
+      belongs_to :order, class_name: "Spree::Order"
```

Comment:
Me and @benjaminwil did some testing and we think it would be a good idea to add a `validates_presence_of :transaction_id` here in addition to the DB-level validation we have.

Reply:
Done :)